doc/sources/contribute-unittest.rst

```diff
@@ -16,11 +16,11 @@ Unit tests are separated into two cases:
 * Graphical unit tests: these need a GL context, and if requested, work via
   image comparison
 
-To be able to run unit tests, you need to install nose
-(http://code.google.com/p/python-nose/), and coverage
-(http://nedbatchelder.com/code/coverage/). You can use easy_install for that::
+To be able to run unit tests, you need to install pytest (https://pytest.org/),
+and coverage (http://nedbatchelder.com/code/coverage/). You can use pip for
+that::
```

Comment:
Is the statement about coverage package still valid or did coverage testing moved over to pytest-cov?

Reply:
dunno